composer.json

```
@@ -10,7 +10,7 @@
         "php": ">=8.1",
         "composer/xdebug-handler": "^3.0",
         "friendsofphp/php-cs-fixer": "^3.12",
-        "nette/utils": "^3.2",
+        "nette/utils": "^3.2 || ^4.0",
```

the vendor is shipped as prefixed vendor, see:

https://github.com/easy-coding-standard/easy-coding-standard/tree/11.3.4/vendor/nette/utils

so it can directly just use v4

```suggestion
        "nette/utils": "^4.0",
```